src/proteus/dswx_hls.py

```diff
@@ -11,15 +11,35 @@
 from ruamel.yaml import YAML as ruamel_yaml
 from osgeo.gdalconst import GDT_Float32
 from osgeo import gdal, osr
+from scipy.ndimage import binary_dilation
+
 from proteus.core import save_as_cog
 
-PRODUCT_VERSION = '0.1'
+PRODUCT_VERSION = '0.5'
```

Comment:
This is not really the product version. It looks like you mean the PROTEOUS sofwtare version. It should not be called "PRODUCT_VERSION" as this should be really defined by a collection of representatives from Algorithm, validation and data system teams. 
I think we agreed that the product version should be an input to the run config file which SDS can populate using an external knowledge.
The software version should be easily derived from our software "e.g., proteou.__version__" or something similar.
Ideally if PCM and PGE could pass the PCM and PGE versions to the SAS through the run config, then we could store all the version infomration we need to track of. To summarize if I'm not mistaken we decided to track multiple versions:

1- product version
2- SAS version 
3- PGE version
4- PCM version
5- ancillary data version

@collinss-jpl @cmarshak can you guys chime in on what is our final conclusion on versioning

Reply:
I agree with what you said Heresh. I don't know the precise nomenclature that you and Scott and others have agreed on.

I do think `PRODUCT_VERSION` should be from the point of the end-user and not be related to the software so should remain at 0.1 until production begins. Just make sure the other version categories (in particular, the proteus software version) are tracked and are saved downstream in the ES database.

Currently, redeployment in SDS wipes all the previously generated datasets. In the future, this will not be the case so being able to facet on the versions will be more important. In particular, being able to differentiate datasets generated by different versions of PROTEUS, in whatever name that is appropriate.

Reply:
@hfattahi, @collinss-jpl , and @cmarshak , thank you for your inputs. I just pushed commit [39cb5ae](https://github.com/nasa/PROTEUS/pull/28/commits/39cb5ae2d742f04022990e604d663c9eeaec1b47) that addresses your comments. I exposed the product version in the runconfig. This field will be used to populate the `PRODUCT_VERSION` in the DSWx-HLS metadata. I also added `SOFTWARE_VERSION` to the DSWx-HLS metadata, which is populated by the PROTEUS version (`v0.5` in this case). Please let me know if this is what you expect and if you have any further comments or concerns.

Reply:
Thanks @gshiroma That is great. @collinss-jpl note that product_version should be populated by PCM. 

The Software version needs to be automated in the final release. I see it hard coded for now which is fine. 

The only part that may still be missing is the SDS version which I think should represent PCM, PGE and ideally the ancillary data. Perhaps that can be added in the final release.